rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -879,7 +881,7 @@ stream {
             {{ end }}
 
             # Pass the extracted client certificate to the auth provider
-            {{ if not (empty $server.CertificateAuth.CAFileName) }}
+            {{ if (not (empty $server.CertificateAuth.VerifyClient)) and (not (eq $server.CertificateAuth.VerifyClient "off")) }}
```

Comment:
You don't need `if (not (empty $server.CertificateAuth.VerifyClient))`.

It wont ever be empty:
https://github.com/kubernetes/ingress-nginx/blob/master/internal/ingress/annotations/authtls/main.go#L33